irrd/conf/__init__.py

```diff
@@ -676,6 +678,19 @@ def get_object_class_filter_for_source(source: str) -> Optional[List[str]]:
         return None
 
 
+def get_nrtm_response_dummy_object_class_for_source(source: str) -> Optional[List[str]]:
```

Comment:
This function is better placed in text.py (or even inlined there) as it's the only place where we use it.

Reply:
Done, thanks